config/config.go

```diff
@@ -107,7 +107,12 @@ func setDefaults() {
 }
 
 func (c *Config) GetDatabaseParams() dbparams.Database {
-	return dbparams.Database{DatabaseURI: viper.GetString("database.connection_string")}
+	return dbparams.Database{
+		URI:                   viper.GetString("database.connection_string"),
```

Comment:
Postgres connection strings can be very non-URI-ish.

Reply:
I'll rename it to ConnectionString to make it clear